_api-reference/index-apis/create-index.md

```diff
@@ -82,7 +82,7 @@ Setting | Description
 index.number_of_shards | The number of primary shards in the index. Default is 1.
 index.number_of_routing_shards | The number of routing shards used to split an index.
 index.shard.check_on_startup | Whether the index's shards should be checked for corruption. Available options are `false` (do not check for corruption), `checksum` (check for physical corruption), and `true` (check for both physical and logical corruption). Default is `false`.
-index.codec | The compression type to use to compress stored data. Available values are `default` (optimizes for retrieval speed) and `best_compression` (optimizes for better compression at the expense of speed, leading to smaller data sizes on disk). For snapshot distributions built with the sandbox feature enabled, `-Dsandbox.enabled=true`, OpenSearch offers a custom-codecs plugin that supports the value `zstd` for Zstandard compression.
+index.codec | The compression type to use to compress stored data. Available values are `default`, which optimizes for retrieval speed, and `best_compression`, which optimizes for better compression at the expense of speed, leading to smaller data sizes on disk. Alternatively, you can use `zstd` and `zstd_no_dict` to optimize for better compression and retrieval speed. `zstd` creates a dictionary with the data being indexed, uses that for compression, and has a minor trade-off on indexing speed. `zstd_no_dict` does not create a dictionary, leading to faster indexing speeds.
```

Comment:
```suggestion
index.codec |  Determines how the index’s stored fields are compressed and stored on the disk. This setting impacts the size of the index shards and the performance of the operations on the index. Available values are:
* `default` - This codec employs the 'LZ4' algorithm with a preset dictionary, which prioritizes performance over compression ratio. It offers faster indexing and search operations when compared with 'best_compression', but may result in larger index/shard sizes. If no codec is provided in the index settings, then 'LZ4'  is used as default algorithm for compression.
* `best_compression` -  This codec utilizes 'zlib' as an underlying algorithm for compression. It achieves high compression ratios resulting in smaller index sizes. However, this may incur additional CPU usage during operations on the index and subsequently may result in high indexing and search latencies. 
* `zstd` - This codec uses the ['Zstandard' compression algorithm](https://github.com/facebook/zstd), which provides a good balance between compression ratio and speed. It provides significant compression comparable to 'best_compression' codec with reasonable CPU usage and improved indexing/search performance comparable to 'default' codec.
* `zstd_no_dict` This codec is similar to 'zstd' but excludes the dictionary compression feature. It provides faster indexing and search operations compared to 'zstd' at the expense of a slightly larger index size.

'zstd' and 'zstd_no_dict' are the new codecs introduced in OpenSearch v2.9.0. They provide an option to configure the compression level as an index setting 'index.codec.compression_level' which is not available for other codecs. 
{: .note}
```